codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/UnionGenerator.kt

```diff
@@ -26,6 +28,12 @@ fun CodegenTarget.renderUnknownVariant() = when (this) {
     CodegenTarget.CLIENT -> true
 }
 
+private val unitShapeId = ShapeId.from("smithy.api#Unit")
+
+internal fun MemberShape.ofTypeUnit(): Boolean {
```

Comment:
This should get moved into `Smithy.kt` in `codegen-core`, and renamed to `isTargetUnit`.

Reply:
`Smithy.kt` looks like the right home for this extension. Thanks. Addressed in de1f5cf.